### PR TITLE
Fix registry secret create

### PR DESCRIPTION
**Pull request by typhoonzero**
Fix https://github.com/PaddlePaddle/cloud/issues/70

### CODE REVIEW

File: paddlecloud/notebook/views.py [hunk 1]
```diff
@@ -206,8 +206,14 @@ def create_user_namespace(username):
                         "data": {
                             "key": encoded
                         }})
+    # create docker registry secret
     registry_secret = settings.JOB_DOCKER_IMAGE.get("registry_secret", None)
-    if registry_secret and registry_secret not in secrets:
+    registry_secret_created = False
```
Comment (Yancey1989) [inline paddlecloud/notebook/views.py]:
```python
secret_names = [item.metadata.name for item in secretes]
if registry_secret and registry_secret in secret_names:
    ...
```
And CephFS secrete can also use the variable `secret_names`, it's a short implementation.

Comment (typhoonzero) [inline paddlecloud/notebook/views.py]: Done

File: paddlecloud/notebook/views.py [hunk 2]
```diff
@@ -185,15 +185,11 @@ def create_user_namespace(username):
             }})
     #create DataCenter sercret if not exists
     secrets = v1api.list_namespaced_secret(user_namespace)
+    secret_names = [item.metadata.name for item in secrets.items]
     for dc, cfg in settings.DATACENTERS.items():
         #create Kubernetes Secret for admin key
         if cfg["fstype"] == "cephfs":
```
Comment (Yancey1989) [inline paddlecloud/notebook/views.py]: It is better for a single line with the two `if` condition ?

Comment (typhoonzero) [inline paddlecloud/notebook/views.py]: Done.

**Pull request merged**